client/routes.js

```diff
@@ -7,11 +7,19 @@ import LoginPage from './components/HomePage/LoginPage';
 import HomePage from './components/HomePage/HomePage';
 import EventPage from './components/Events/index';
 import SocialClubPage from './components/SocialClub/SocialClubPage';
+import EnsureLoggedIn from "./components/common/EnsureLoggedIn";
+import NotFound from "./components/common/NotFound";
+
 
 export default
   <Route path="/" component={App}>
     <IndexRoute component={LoginPage} />
-  	<Route path="/home" component={HomePage} />
-  	<Route path="/clubs/:id" component={SocialClubPage} />
-    <Route path="/clubs/:club_id/events/:id" component={EventPage} />
+
+    <Route component={EnsureLoggedIn}>
```

Comment:
Oh! I see what the `EnsureLoggedIn` component does now. But I feel we can give it a better name. As opposed to `EnsureLoggedIn`